### DISCUSSION TITLE
How to create tractogram from a multi-shell data for RecoBundles

### DISCUSSION THREAD

**Discussion by miladseyfi** [dipy/dipy#2366] [Tractography / Tractometry]
Hello dear developers and users
I have a multi-shell diffusion data and i want to do automatic bundle extraction using RecoBundles.
In the documentation in the first step tractogram of both target and atlas should load.
I could not found clear explanation to build this tractogram from a multi-shell data .
If any body has solution I appreciate that so much.

**Comment by skoudoro** [comment]
Hi @miladseyfi,

Thank you for asking. It does not look like an issue, so I am moving this demand to forum / discussions.

You can follow [this tutorial](https://dipy.org/documentation/1.4.0./examples_built/tracking_pft/#example-tracking-pft) to create initial tractogram or pick a [fiber tracking tutorial](https://dipy.org/tutorials/) in documentation.

@BramshQamar, Can you provide additional information? Thanks
